TrainingTask/MyCar.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TrainingTask
+{
+    public class MyCar : Drawer
+    {
+        private TrackField trackField = new TrackField();
+
+        private readonly char carSymbol;
+
+        public static int MyCarPosition { get; set; }
```

Comment:
Don't see any reasons for static member.

Reply:
-1 to source code quality.

Reply:
Static status allows threads to use the same variable (MyCarPosition) and change variable dynamically. Isn't it?